index.src.html

```diff
@@ -332,22 +332,46 @@ <h3 id="persistent-origin-specific-state">
       mechanisms will form a persistent identifier by offering users the
       ability to wipe out the data contained in these types of storage.
 
-  *   Fingerprinting is a technique establishing a unique identifier based on
-      the capabilities of specific feature alone (e.g. Canvas), or combining a
-      number of features. Specifications and user agents should treat the risk
-      of fingerprinting by carefully considering the surface of available
-      information, and the relative differences between software and hardware
-      stacks. Sometimes reducing fingerprintability may be revolve to specific
-      operational considerations, for example providing information in same
-      order (i.e. list of fonts), but sometimes this is not so simple.
-
-  *   Some readout information that is subject to change may still act as a
-      short-term identifier, and possibly introduce a risk of misuse (examples:
-      <a href="https://eprint.iacr.org/2015/616">Leaking Battery</a>,
-      <a href="https://blog.lukaszolejnik.com/battery-status-not-included-assessing-privacy-in-w3c-web-standards/">
-      Battery Status Not Included</a>). Good example may be the readout of
-      ambient light level [AMBIENT-LIGHT], or battery [BATTERY-STATUS-API]
+  <h3 id="underlying-platform-data">
+    Does this specification expose information to an origin that originates
```

Comment:
Quite a long question.

Reply:
Maybe "Does this specification expose information that originates from the underlying platform that is consistent across origins, including user configurations or sensors?" ?

Reply:
"Does this specification expose information that originates from the underlying platform?"

And in the text: "If so, might the offered information be consistent across origins? This includes information relating to the user configuration, sensors, communication methods, etc". Something like that.

Reply:
See d9b23e18cdbc9efeee780fb0094a878c7c86f8ec.

Reply:
See 
https://github.com/w3ctag/security-questionnaire/pull/41/commits/d9b23e18cdbc9efeee780fb0094a878c7c86f8ec.